db/migrate/20230517233628_add_hideouts_table.rb

```diff
@@ -0,0 +1,8 @@
+class AddHideoutsTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :hideouts do |t|
+      t.string :name, null: false, index: {unique: true}
```

Comment:
Does the `index: {unique: true}` on line 4 mean that two hideouts cannot have the same name? Don't think we want that, but I might have misundertood the code.